core/src/main/java/io/grpc/InternalStatus.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc;
+
+/**
+ * Accesses internal data.  Do not use this.
+ */
+@Internal
+public final class InternalStatus {
```

Comment:
I'd honestly be happy with these in just the `io.grpc.Internal` class. I'm not sure we need a ton of different Internal classes.